src/components/generic/generic-dialog.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
   positiveButtonText?: string;
   cancelButtonText?: string;
   onClose: () => void;
-  onCancel: () => void;
+  onCancel?: () => void;
```

Comment:
Why has this been changed?

Reply:
So the cancel button could be removes when wanted